app/controllers/legacy_routes/handler.rb

```diff
@@ -5,13 +5,13 @@
 #  See the COPYING file at the top-level directory or at
 #  https://github.com/puzzle/cryptopus.
 
-class LegacyRoutes::Locale
+class LegacyRoutes::Handler
```

Comment:
name this RedirectUrl instead of Handler

Reply:
there's no unit test for this class ?!
TDD ... make sure you handle all urls in this unit test first